README.md

```diff
@@ -88,13 +88,11 @@ $ coverage report -m
 #### GET  '/api/rand_exercise'
 
 ```
-#REQUEST BODY
-duration options are 5:00, 7:00, 10:00
-category options are SOMATIC, MOVEMENT, MEDITATION
-{
-    "duration": "5:00",
-    "category": "SOMATIC"
-}
+#QUERY PARAMS
+* Required
+
+* duration = 5:00, 7:00, 10:00
```

Comment:
Thank you for altering this for us!

Reply:
If you wouldn't mind updating the documentation at some point too, that would be great!